### ISSUE TITLE
请问预训练时怎么划分训练集和测试集呢？How to split training data and test data during pretraining?

### ISSUE THREAD

**Issue by WHQ1111** [google-research/bert#974]
I'm wondering how to split the training set and test set during pretraining. Are there someone kindly tell me?

**Comment by visonabc** [comment]
from sklearn.model_selection import train_test_split


train,test = train_test_split(train, test_size=0.01)
